console/frontend/src/main/frontend/src/app/views/jdbc/jdbc-execute-query/quick-submit-form.directive.spec.ts

```diff
@@ -34,13 +36,13 @@ describe('QuickSubmitFormDirective', () => {
     );
   });
 
-  it('emits event when ctrl+enter is pressed', () => {
+  it('emits event when ctrl+enter is pressed', fakeAsync(() => {
```

Comment:
Could you check if this is still needed?

Reply:
It wasnt needed anymore